src/httpbis/index.ts

```diff
@@ -93,6 +96,60 @@ export function buildSignatureInputString(componentNames: Component[], parameter
     }).join('')}`
 }
 
+export function parseSignatureInputString(signatureInput: string): { [signatureName: string]: ParsedSignatureInput } {
+
+    return signatureInput.split[','].reduce((signatureInputs, signatureInputString) => {
+
+        const [signatureName, signatureInputValue] = signatureInputString.trim().split['=']

```

Comment:
How naive is this split logic? Can the signature input values  contain `=` symbols? Would they be escaped?

I'm worried that we could have strings with multiple `=` and this may not correctly parse the values.

Reply:
Very naive. All of the parsing will need to be made more robust but this does give you a sense of the flow.

All of these strings follow the rules in https://datatracker.ietf.org/doc/html/rfc8941. I'll ask the editors if they know of any good parsers that are already available.

Reply:
To be more specific https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-message-signatures-11#section-4.1 defines both the signature input and signature as Dictionaries so we'd need to follow https://datatracker.ietf.org/doc/html/rfc8941#section-4.2.2 to parse these first.